deps/wazuh_testing/wazuh_testing/modules/aws/data_generator.py

```diff
@@ -94,22 +103,258 @@ def get_data_sample(self) -> dict:
                     'sharedEventID': str(uuid4())
                 }
             ]
-        }
+        })
+
+
+class VPCDataGenerator(DataGenerator):
+    BASE_PATH = f'{cons.AWS_LOGS}/{cons.RANDOM_ACCOUNT_ID}/{cons.VPC_FLOW_LOGS}/{cons.US_EAST_1_REGION}/'
```

Comment:
F-strings must use double quotes as long as it is possible.

Reply:
I need to cherry-pick some functionality included in https://github.com/wazuh/wazuh-qa/pull/3699 . This correction has already been fixed in 258068a66a1a5bf055be4574674df62e8841f9ac and will be included in [3333](https://github.com/wazuh/wazuh-qa/tree/3333-aws-integration-tests)